### ISSUE TITLE
bucket name is required but README says it is not, results in error

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-aws-cloudformation-v2#11]
## Do you already have an issue opened with F5 support?
GitHub Issues are consistently monitored by F5 staff, but should be considered as best-effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open a case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
The readme inputs specify that the s3bucketname and region are optional. The column for required says "no". However if you leave them blank then you receive error.

My use case = I do not need to use s3 buckets as I'm using template as-is and public repo. Doing quick tests. It appears that a bucket is still needed as a pre-req.

error
```
An error occurred (ValidationError) when calling the CreateStack operation: Parameter s3BucketName failed to satisfy constraint: S3 bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-).
```

What I tried (snippet from parameters file)
```
  {
    "ParameterKey": "s3BucketName",
    "ParameterValue": ""
  },
  {
    "ParameterKey": "s3BucketRegion",
    "ParameterValue": ""
  },
```

## Template
https://github.com/F5Networks/f5-aws-cloudformation-v2/tree/main/examples/quickstart

## Severity Level
3

**Comment by JeffGiroux** [comment]
Quick look at the quickstart.yml file in the repo and it does indeed say the parameter for bucket name is required. Can you please match the required/optional items in the README to make sure instructions are accurate.

Example in quickstart.yml
```
    Description: 'REQUIRED - S3 bucket name for the modules. S3 bucket name can include
      numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start
      or end with a hyphen (-).'
```

And this is worded differently than the README inputs...
```
| s3BucketRegion | No | AWS Region which contains the S3 Bucket containing templates |
| s3BucketName | No | S3 bucket name for the modules. S3 bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-). |
```

**Comment by JeffGiroux** [comment]
Update, using the provided as-is values will deploy example quickstart just fine. However there needs to be further documentation around using customer s3 buckets and also a note that tells user that proper permissions and IAM roles are required. As it stands now, I created a private bucket, used that bucket, then I got an access error denied trying to run the rest of the template as-is. 

As a general user of this template for first time, I would expect after reading the basic instructions that I should be able to point to a bucket...and then the access template correctly configures permissions and IAM roles. This doesn't appear to be the case though.

Whatever the intent...please make sure README instructions for pre-req are clear regarding how the template uses a custom s3 bucket and what permissions are required.

**Comment by shyawnkarim** [comment]
Thanks for reporting this. I've created a documentation update for this issue, internal ID ESECLDTPLT-2991, to get our README updated.

**Comment by shyawnkarim** [comment]
Closing.

This ticket was included with [Release 2.2.0.0](https://github.com/F5Networks/f5-aws-cloudformation-v2/releases/tag/v2.2.0.0).
